[2023-09-27 20:25:24,304] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.100.183:5001 (Press CTRL+C to quit) - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:25:24,310] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:25:30,237] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:25:30] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:25:30,264] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:25:30] "GET /openapi/ HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:25:30,572] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:25:30] "[36mGET /openapi/static/images/rapidoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:25:30,576] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:25:30] "[36mGET /openapi/static/images/redoc.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:25:30,589] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:25:30] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:10,986] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:10] "OPTIONS /cliente?name=Joao HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:11,026] WARNING del_cliente() L133  Erro ao deletar cliente #'Joao', Cliente não encontrado na base - call_trace=C:\Users\mc92_\Desktop\MV2_BackEndAvançado\componente_c_microsservico\app.py L133 
[2023-09-27 20:26:11,031] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:11] "[33mDELETE /cliente?name=Joao HTTP/1.1[0m" 404 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:35,946] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:35] "POST /cliente HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:51,534] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:51] "GET /openapi/swagger HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:51,581] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:51] "[36mGET /openapi/static/css/swagger-ui.css HTTP/1.1[0m" 304 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:51,603] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:51] "[36mGET /openapi/static/js/swagger-ui-bundle.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:51,605] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:51] "[36mGET /openapi/static/js/swagger-ui-standalone-preset.js HTTP/1.1[0m" 304 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:51,857] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:51] "GET /openapi/openapi.json HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:26:52,058] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:26:52] "[36mGET /openapi/static/images/swagger.svg HTTP/1.1[0m" 304 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:27:20,191] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:27:20] "POST /cliente HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:27:35,471] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:27:35] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:28:07,698] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:28:07] "DELETE /cliente?name=Pedro HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:28:08,588] WARNING del_cliente() L133  Erro ao deletar cliente #'Pedro', Cliente não encontrado na base - call_trace=C:\Users\mc92_\Desktop\MV2_BackEndAvançado\componente_c_microsservico\app.py L133 
[2023-09-27 20:28:08,589] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:28:08] "[33mDELETE /cliente?name=Pedro HTTP/1.1[0m" 404 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:28:40,793] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:28:40] "OPTIONS /cliente?name=Joao HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:28:40,802] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:28:40] "DELETE /cliente?name=Joao HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:29:29,172] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:29:29] "POST /cliente HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:29:44,635] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:29:44] "POST /cliente HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:29:55,163] WARNING get_cliente() L102  Erro ao buscar cliente 'Ex: 1', Cliente não encontrado na base - call_trace=C:\Users\mc92_\Desktop\MV2_BackEndAvançado\componente_c_microsservico\app.py L102 
[2023-09-27 20:29:55,165] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:29:55] "[33mGET /cliente?name=Ex%3A%201 HTTP/1.1[0m" 404 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:29:58,435] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:29:58] "GET /cliente?name=2 HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:30:04,134] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:30:04] "GET /clientes HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:30:15,427] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:30:15] "PUT /cliente HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:30:16,334] WARNING add_revisao_put() L164  Erro ao adicionar cliente 'George Smith', Cliente já adicionado - call_trace=C:\Users\mc92_\Desktop\MV2_BackEndAvançado\componente_c_microsservico\app.py L164 
[2023-09-27 20:30:16,338] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:30:16] "[31m[1mPUT /cliente HTTP/1.1[0m" 409 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:30:28,069] WARNING add_revisao_put() L164  Erro ao adicionar cliente 'Pedro', Cliente já adicionado - call_trace=C:\Users\mc92_\Desktop\MV2_BackEndAvançado\componente_c_microsservico\app.py L164 
[2023-09-27 20:30:28,073] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:30:28] "[31m[1mPUT /cliente HTTP/1.1[0m" 409 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:30:42,504] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:30:42] "PUT /cliente HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2023-09-27 20:30:51,829] INFO _log() L224  127.0.0.1 - - [27/Sep/2023 20:30:51] "DELETE /cliente?name=Marcos HTTP/1.1" 200 - - call_trace=C:\Users\mc92_\AppData\Local\Programs\Python\Python310\lib\site-packages\werkzeug\_internal.py L224 
[2024-02-18 10:23:30,074] INFO _log() L224   * Running on all addresses (0.0.0.0)
   WARNING: This is a development server. Do not use it in a production deployment.
 * Running on http://127.0.0.1:5001
 * Running on http://192.168.100.183:5001 (Press CTRL+C to quit) - call_trace=C:\Users\mc92_\Desktop\MVP2_BACKEND_REFORMULADO (PORTUGUES)\componente_c_microsservico\env\lib\site-packages\werkzeug\_internal.py L224 
[2024-02-18 10:23:30,078] INFO _log() L224   * Restarting with stat - call_trace=C:\Users\mc92_\Desktop\MVP2_BACKEND_REFORMULADO (PORTUGUES)\componente_c_microsservico\env\lib\site-packages\werkzeug\_internal.py L224 
[2024-02-18 10:23:33,754] INFO _log() L224  127.0.0.1 - - [18/Feb/2024 10:23:33] "[32mGET / HTTP/1.1[0m" 302 - - call_trace=C:\Users\mc92_\Desktop\MVP2_BACKEND_REFORMULADO (PORTUGUES)\componente_c_microsservico\env\lib\site-packages\werkzeug\_internal.py L224 
